app/grandchallenge/core/templates/grandchallenge/partials/navbar.html

```diff
@@ -31,6 +31,14 @@
                         Reader Studies
                     </a>
                 </li>
+                {% for overview_page in overview_pages %}
+                    <li class="nav-item">
+                        <a class="nav-link {% if request.resolver_match.view_name == 'overview-pages:detail' %}active{% endif %}"
```

Comment:
Wouldn't this make all nav items active on all overview pages?

Reply:
Ah yes, we don't do anything with active in the navbar anyway so I will remove it

Reply:
Actually kept it by looking up slug too. Can't use the url checks as we have this complex url reversal for subdomains.